src/command_modules/azure-cli-ml/azure/cli/command_modules/ml/_params.py

```diff
@@ -71,3 +71,6 @@
 register_cli_argument('ml env setup', 'status', options_list=('-s', '--status'), metavar='deploymentId', help='Check the status of an ongoing deployment.', required=False)
 register_cli_argument('ml env setup', 'name', options_list=('-n', '--name'), metavar='envName', help='The name of your Azure ML environment (1-20 characters, alphanumeric only).', required=False)
 register_cli_argument('ml env setup', 'kubernetes', options_list=('-k', '--kubernetes'), action='store_true', help='Sets up a new Kubernetes cluster.', required=False)
+register_cli_argument('ml env setup', 'service_principal', options_list=('-p', '--service-principal'), help='Service principal to use for configuring ACS cluster.', required=False)
+register_cli_argument('ml env setup', 'client_secret', options_list=('-c', '--client-secret'), help='Client secret associated with service principal.', required=False)
+register_cli_argument('ml env setup', 'tenant', options_list=('-t', '--tenant'), help='Tenant associated with service principal.', required=False)
```

Comment:
Change these to match the output of 'az ad sp create-for-rbac'. This is a little confusing.